rosdep/python.yaml

```diff
@@ -2441,7 +2398,12 @@ python-vtk:
   arch: [vtk]
   debian: [python-vtk]
   fedora: [vtk-python]
-  ubuntu: [python-vtk]
+  ubuntu:
+    trusty: [python-vtk]
+    utopic: [python-vtk]
+    vivid: [python-vtk]
+    wily: [python-vtk]
+    xenial: [python-vtk6]
```

Comment:
I believe this is what should happen, given that `python-vtk` and `python-vtk6` are in conflict, and the base `libvtk` rosdep in Xenial points to `libvtk6-dev`, which pulls in `python-vtk6`:

https://github.com/ros/rosdistro/blob/ec21789bdf300a7ec4e21a1acbd391caa9b97447/rosdep/base.yaml#L2749
http://packages.ubuntu.com/xenial/libvtk6-dev